src/utilities/theme/utils.ts

```diff
@@ -14,7 +14,7 @@ export function buildCustomProperties(
 ): CustomPropertiesLike {
   return globalTheming
     ? {
-        ...buildColors(themeConfig),
+        ...customPropertyTransformer(buildColors(themeConfig)),
```

Comment:
I’d like to move `customPropertyTransformer` back to `buildColors` and place a layer of indirection a bit deeper. I know the goal is to have a function that returns a more standard json-ish blob since we are looking at setting these values in context as well, but I think for now it is nice to have the symmetry between `buildColors` and `buildLegacyColors`, super easy to parse and reason about when it is:

 `return globalTheming ? buildColors(themeConfig) : buildLegacyColors(themeConfig);`